xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -544,6 +548,71 @@ rebuild_master:
         project: home:Admin
         package: ctris</screen>
         </sect4>
+
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.set_flags">
+          <title>Set Flags for Projects, Packages, Repositories or Architectures</title>
+
+          <para>Providing the flag <emphasis>build</emphasis>, the status <emphasis>enable</emphasis> and the project <emphasis>home:Admin</emphasis> will enable <emphasis>all</emphasis> builds:</para>
+
+          <itemizedlist>
+            <listitem>
+              <para>for the <emphasis>home:Admin:$MY_SCM_ORG:$MY_SCM_PROJECT:PR-$MY_PR_NUMBER</emphasis> project
+              when the webhook event is a pull request.</para>
+            </listitem>
+            <listitem>
+              <para>for the <emphasis>home:Admin</emphasis> project when the webhook event is a push.</para>
+            </listitem>
+          </itemizedlist>
+
+          <para>Please note that the provided project has to be a project which was a target project from a previous step like <emphasis>link_package</emphasis>
+          or <emphasis>branch_package</emphasis>. Same for the package, it has to be branched/linked in a previous step.</para>
```

Comment:
Not necessarily true for push events right? I think what you are trying to say is that project and package should exist?

Reply:
Would you remove this paragraph? I think it's already clear that the project and package have to exist... otherwise why would someone even want to apply a flag to something which doesn't even exist.

Reply:
Let me know what you think.